src/App.tsx

```diff
@@ -11,27 +12,44 @@ interface itemsTypes {
   isEmpty: boolean;
 }
 
+const initialItems = [
+  {
+    todoId: 1,
+    content: "Item 1",
+    isChecked: false,
+    isEmpty: false,
+  },
+  {
+    todoId: 2,
+    content: "Item 2",
+    isChecked: false,
+    isEmpty: false,
+  },
+  {
+    todoId: 3,
+    content: "Item 3",
+    isChecked: false,
+    isEmpty: false,
+  },
+];
+
 const App = () => {
-  const [items, setItems] = useState<itemsTypes[]>([
-    {
-      todoId: 1,
-      content: "Item 1",
-      isChecked: false,
-      isEmpty: false,
-    },
-    {
-      todoId: 2,
-      content: "Item 2",
-      isChecked: false,
-      isEmpty: false,
-    },
-    {
-      todoId: 3,
-      content: "Item 3",
-      isChecked: false,
-      isEmpty: false,
-    },
-  ]);
+  const [items, setItems] = useState<itemsTypes[]>(initialItems || []);
```

Comment:
There is no need of the conditional empty array `|| []` here.
The code `initialItems || []` means that you want to set the initial value of the state to an empty array if there is no initialItems. However, you will always have the `initialValue` as it is an object you defined in the beginning of the file.
```suggestion
  const [items, setItems] = useState<itemsTypes[]>([]);
```